ooiui/templates/common/opLog.html

```diff
@@ -113,6 +113,7 @@
   views: {
   },
   start: function() {
+    this.login.fetch({async:false});
```

Comment:
`{async:false}` is ok here because it doesn't actually make an AJAX request, it just checks the browser cookies.